src/CodeGen_Internal.cpp

```diff
@@ -464,7 +464,11 @@ std::unique_ptr<llvm::TargetMachine> make_target_machine(const llvm::Module &mod
                                                 mcpu, mattrs,
                                                 options,
                                                 llvm::Reloc::PIC_,
+#if LLVM_VERSION < 60
```

Comment:
i think it might have moved to 60

commit 89fb1f68e327643614e9b26409c2f773336578ea
Author: Hans Wennborg <hans@hanshq.net>
Date:   Wed Jul 19 12:19:01 2017 +0000

    Update trunk version to 6.0.0svn